src/api/location/get/getSityandCountry.js

```diff
@@ -0,0 +1,17 @@
+//imports
```

Comment:
Decomposition is good way to control your  app logic, BUT...I suppose that you can create posts.api and add logic methods in there (get, post, patch, puth)

'/api/get/getSityandCountry.js' is too complicated way for this app